diff --git a/node_modules/@korsolutions/react-native-joystick/lib/index.js b/node_modules/@korsolutions/react-native-joystick/lib/index.js
index ca3d80f..9c3548f 100644
--- a/node_modules/@korsolutions/react-native-joystick/lib/index.js
+++ b/node_modules/@korsolutions/react-native-joystick/lib/index.js
@@ -1,7 +1,7 @@
-import React, { useState, useCallback, useMemo } from "react";
-import { View, StyleSheet } from "react-native";
-import * as utils from "./utils";
+import { useCallback, useMemo, useState } from "react";
+import { StyleSheet, View } from "react-native";
 import { Gesture, GestureDetector } from "react-native-gesture-handler";
+import * as utils from "./utils";
 export const ReactNativeJoystick = ({ onStart, onMove, onStop, color = "#000000", radius = 150 }) => {
     const wrapperRadius = radius;
     const nippleRadius = wrapperRadius / 3;
@@ -71,7 +71,7 @@ export const ReactNativeJoystick = ({ onStart, onMove, onStop, color = "#000000"
                 type: "start",
             });
     };
-    const panGesture = Gesture.Pan().onStart(handleTouchStart).onEnd(handleTouchEnd).onTouchesMove(handleTouchMove);
+    const panGesture = Gesture.Pan().runOnJS(true).onStart(handleTouchStart).onEnd(handleTouchEnd).onTouchesMove(handleTouchMove);
     const styles = useMemo(() => StyleSheet.create({
         wrapper: {
             width: 2 * radius,
